src/documents/IoCReferences.md

```diff
@@ -193,7 +193,6 @@ fluid.defaults("fluid.videoPlayer.controllers", {
             container: "{controllers}.dom.scrubberContainer",
             options: {
                 model: "{controllers}.model",
-                applier: "{controllers}.applier",
```

Comment:
Why don't we specify the applier anymore? This might be a change that should be explained in the "API changes from 1.5 to 2.0" document.